File: pinyougou-parent/pinyougou-manager-web/src/main/webapp/js/service/brandService.js
/*创建服务层代码*/
app.service("brandService",function ($http) {
    this.findAll = function () {
        return $http.get("../brand/findAll.do");
    };
    this.findPage = function (pageNum, pageSize) {
        return $http.get('../brand/pageFind.do?pageNum='+pageNum+'&pageSize='+pageSize);
    };
    this.add = function (entity) {
        return $http.post('../brand/add.do',entity)
    };
    this.update = function (entity) {
        return $http.post('../brand/update.do',entity)
    };
    this.findOne = function (id) {
        return $http.post('../brand/findOne.do?id='+id);
    };
    this.del = function (selectIds) {
        return $http.get("../brand/delete.do?ids="+selectIds);
    };
    this.search = function (pageNum, pageSize,searchEntity) {
        return $http.post('../brand/search.do?pageNum='+pageNum+'&pageSize='+pageSize,searchEntity);
    };
    this.selectOptionList = function () {
        return $http.get('../brand/selectOptionList.do');
    }
});

File: pinyougou-parent/pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import entity.PageResult;
import entity.Result;

import java.util.List;
import java.util.Map;

/**
 * 品牌接口
 */

public interface BrandService {
    /**
     * 查询所有品牌
     * @return
     */
    public List<TbBrand> findAll();

    /**
     * 分页查询
     * @param pageNum
     * @param pageSize
     * @return
     */
    public PageResult pageFind(int pageNum,int pageSize);

    /**
     * 增加一个品牌
     * @param tbBrand
     */
    public void add(TbBrand tbBrand);

    /**
     * 查找一个实体
     * @param id
     * @return
     */
    public TbBrand findOne(Long id);

    /**
     * 修改一个品牌
     * @param tbBrand
     * @return
     */
    public void update(TbBrand tbBrand);

    /**
     * 删除一个品牌
     * @param id
     */
    public void delete(Long id);

    /**
     * 分页
     * @param tbBrand
     * @param pageNum
     * @param pageSize
     * @return
     */
    public PageResult findPage(TbBrand tbBrand,int pageNum,int pageSize);

    /**
     * 查询下拉列表
     * @return
     */
    public List<Map> selectOptionList();
}
